--- LuaJIT-a4f56a459a588ae768801074b46ba0adcfb49eb1/src/Makefile.orig	2025-09-16 15:42:57.091915947 +0800
+++ LuaJIT-a4f56a459a588ae768801074b46ba0adcfb49eb1/src/Makefile	2025-09-16 15:42:58.922898907 +0800
@@ -98,7 +98,7 @@
 # enabled by default. Some other features that *might* break some existing
 # code (e.g. __pairs or os.execute() return values) can be enabled here.
 # Note: this does not provide full compatibility with Lua 5.2 at this time.
-#XCFLAGS+= -DLUAJIT_ENABLE_LUA52COMPAT
+XCFLAGS+= -DLUAJIT_ENABLE_LUA52COMPAT
 #
 # Disable the JIT compiler, i.e. turn LuaJIT into a pure interpreter.
 #XCFLAGS+= -DLUAJIT_DISABLE_JIT
