--- a/autosetup/sqlite-config.tcl       2025-07-30 19:45:35.000000000 +0000
+++ a/autosetup/sqlite-config.tcl       2025-09-17 04:19:13.849975768 +0000
@@ -1955,7 +1955,7 @@
       }
     }
     if {![file-isexec $with_tclsh]} {
-      proj-warn "Cannot find a usable tclsh (tried: $tryThese)
+      proj-warn "Cannot find a usable tclsh (tried: $tryThese)"
     }
   }
   define TCLSH_CMD $with_tclsh
