--- boringssl-0.20251002.0/CMakeLists.txt
+++ boringssl-0.20251002.0/CMakeLists.txt
@@ -803,6 +803,6 @@ if(INSTALL_ENABLED)
   install(EXPORT OpenSSLTargets
           FILE OpenSSLTargets.cmake
           NAMESPACE OpenSSL::
-          DESTINATION lib/cmake/OpenSSL)
-  install(FILES cmake/OpenSSLConfig.cmake DESTINATION lib/cmake/OpenSSL)
+          DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/OpenSSL)
+  install(FILES cmake/OpenSSLConfig.cmake DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/OpenSSL)
 endif()

