Index: cmake-4.1.0/Modules/FindRuby.cmake
===================================================================
--- cmake-4.1.0.orig/Modules/FindRuby.cmake
+++ cmake-4.1.0/Modules/FindRuby.cmake
@@ -285,14 +285,9 @@
   _RUBY_CONFIG_VAR("sitearchdir" Ruby_SITEARCH_DIR)
   _RUBY_CONFIG_VAR("sitelibdir" Ruby_SITELIB_DIR)
 
-  # vendor_ruby available ?
-  execute_process(COMMAND ${Ruby_EXECUTABLE} -r vendor-specific -e "print 'true'"
-                  OUTPUT_VARIABLE Ruby_HAS_VENDOR_RUBY ERROR_QUIET)
-
-  if (Ruby_HAS_VENDOR_RUBY)
-    _RUBY_CONFIG_VAR("vendorlibdir" Ruby_VENDORLIB_DIR)
-    _RUBY_CONFIG_VAR("vendorarchdir" Ruby_VENDORARCH_DIR)
-  endif ()
+  # vendor_ruby
+  _RUBY_CONFIG_VAR("vendorlibdir" Ruby_VENDORLIB_DIR)
+  _RUBY_CONFIG_VAR("vendorarchdir" Ruby_VENDORARCH_DIR)
 
   # save the results in the cache so we don't have to run ruby the next time again
   set(_Ruby_EXECUTABLE_LAST_QUERIED "${Ruby_EXECUTABLE}" CACHE INTERNAL "The ruby executable last queried for version and path info")
