diff --git a/libtextstyle/gnulib-local/lib/iconv-ostream.oo.c b/libtextstyle/gnulib-local/lib/iconv-ostream.oo.c
index 60cb174..5c94cff 100644
--- a/libtextstyle/gnulib-local/lib/iconv-ostream.oo.c
+++ b/libtextstyle/gnulib-local/lib/iconv-ostream.oo.c
@@ -245,7 +245,7 @@ iconv_ostream::write_mem (iconv_ostream_t stream, const void *data, size_t len)
 }
 
 static void
-iconv_ostream::flush (iconv_ostream_t stream)
+iconv_ostream::flush (iconv_ostream_t stream, ostream_flush_scope_t scope)
 {
   abort ();
 }
diff --git a/libtextstyle/lib/iconv-ostream.oo.c b/libtextstyle/lib/iconv-ostream.oo.c
index 60cb174..5c94cff 100644
--- a/libtextstyle/lib/iconv-ostream.oo.c
+++ b/libtextstyle/lib/iconv-ostream.oo.c
@@ -245,7 +245,7 @@ iconv_ostream::write_mem (iconv_ostream_t stream, const void *data, size_t len)
 }
 
 static void
-iconv_ostream::flush (iconv_ostream_t stream)
+iconv_ostream::flush (iconv_ostream_t stream, ostream_flush_scope_t scope)
 {
   abort ();
 }
